specificatie/genereervariant/openapi.yaml

```diff
@@ -519,6 +508,162 @@ paths:
                   to a temporary overloading or maintenance of the server.
                 instance: https://datapunt.voorbeeldgemeente.nl/api/v1/resourcenaam?parameter=waarde
                 code: notAvailable
+  /kadastraalonroerendezaken/burgerservicenummer:
```

Comment:
Is dit een resource? De resource is nog steeds /kadastraalonroerendezaken toch? Je zoekt immers kadastraalonroerendezaken, ongeacht de parameter waarmee je zoekt. De parameter is een middel om mee te zoeken, en geen onderdeel van hetgeen wordt gezocht.

/kadastraalonroerendezaken/burgerservicenummer is geen resourcenaam.

Wij vermoeden dat dit endpoint in het leven is geroepen om eea te isoleren van de het zoeken op rechtspersoon en kadasterpersonen. Maar ook de identificatie van kadasternatuurlijkepersonen is een identificerend gegeven. 
Voorstel: 
Post/kadastraalonroerendezaken
description: endpoint voor zoeken met BSN en KadasterNatuurlijkepersonen
Get/kadastraalonroerendezaken 
description: endpoint voor zoeken met rechtspersonen enz.

Reply:
Goed punt. Zal het endpoint wijzigen naar `/kadastraalonroerendezaken`

Reply:
> Is dit een resource? De resource is nog steeds /kadastraalonroerendezaken toch? Je zoekt immers kadastraalonroerendezaken, ongeacht de parameter waarmee je zoekt. De parameter is een middel om mee te zoeken, en geen onderdeel van hetgeen wordt gezocht.
> 
> /kadastraalonroerendezaken/burgerservicenummer is geen resourcenaam.
> 
> Wij vermoeden dat dit endpoint in het leven is geroepen om eea te isoleren van de het zoeken op rechtspersoon en kadasterpersonen. Maar ook de identificatie van kadasternatuurlijkepersonen is een identificerend gegeven. Voorstel: Post/kadastraalonroerendezaken description: endpoint voor zoeken met BSN en KadasterNatuurlijkepersonen Get/kadastraalonroerendezaken description: endpoint voor zoeken met rechtspersonen enz.

Beetje late reactie, maar dat is vakantie-gerelateerd. Ik weet ook niet of dit modterd na de maaltijd is. 
Is dit wel de verstandigste keuze? Bij deze naamgeving zou ik verwachten dat ik met de Post /kadastraalonroerendezaken een onroerendezaak kan vastleggen. Dat is niet at we bedoelen. 
Op het moment dat je een POST gebruikt om een zoek-actie te doen heb je het eigenlijk niet meer over een REST-georienteerde API. Het lijkt mij dan ook verstandiger om hier een ander endpoint voor te gebruiken en niet hetzelfde endpoint met een andere operatie. Volgens de API-strategie zou een dergelijk endpoint ook met een inderscore moeten beginnen. Een beter keuze lijkt mij POST /_zoekkadastraalonroerendezaakmetbsn.